tapeout/src/test/resources/macros/lib-2048x16-n28.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "type": "sram",
+    "name": "vendor_sram_16",
```

Comment:
Perhaps all the "name" fields should be consistent? I'm not sure what the benefit of having only some of them be different.